Carta.py

```diff
@@ -386,57 +340,85 @@ def opponentMove(self):
         for i in range(len(self.grabbingCardsInPlay)):
             if ((self.grabbingCardsInPlay[i].getLastWord() == self.curReadingCard.getLastWord()) and \
                 (takeCorrectCard)):
-                self.info.opponentGrabbingCard = self.grabbingCardsInPlay[i]
+                self.info.oppoGrabCardLastWord = self.grabbingCardsInPlay[i].getLastWord()
                 break
             elif ((self.grabbingCardsInPlay[i].getLastWord() != self.curReadingCard.getLastWord()) and \
                   (not takeCorrectCard)):
-                self.info.opponentGrabbingCard = self.grabbingCardsInPlay[i]
+                self.info.oppoGrabCardLastWord = self.grabbingCardsInPlay[i].getLastWord()
                 break
 
         self.info.opponentTime = self.info.startTime + self.parameters.opponentTimeForStupidAI
-        self.info.opponentMoved = True
+        self.info.oppoGrabbedCard = True
+
+        if ((takeCorrectCard) and (self.info.oppoGrabCardLastWord != "")):
+            print("opponent take correct card")
```

Comment:
wrap every print under debug mode
modify every print statement

For example:
print("Carta.py: opponentGrabsCard: opponent take correct card")